ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -301,6 +301,14 @@ if [[ "$T5CI_VERSION" == "4.15" ]] || [[ "$T5CI_VERSION" == "4.16" ]]; then
     # git version 1.8 doesn't work well with forked repositories, requires a specific branch to be set
     sed -i "s@https://github.com/openshift/metallb-operator.git@https://github.com/openshift/metallb-operator.git\n        branch = main@" .gitmodules
     make init-git-submodules
+    checkout_submodules(){
```

Comment:
Will not make a difference here but as a best practice: Instead of defining checkout_submodules within the conditional block, define it before the if statement to ensure it's accessible throughout the script.